app/controllers/components/course/users_component.rb

```diff
@@ -4,6 +4,7 @@ class Course::UsersComponent
   sidebar do
     [
       {
+        key: :users,
```

Comment:
We already have some docs in `controllers//course/controller.rb`, but not that obvious to find.